File: Örnek-6.11/Java/yatayatıs.java
package algoritma;
import java.util.Scanner;


public class yatayatıs {

	public static void main(String[] args) {
		double h,vo,t,x,vy,v;
		
		Scanner scan=new Scanner(System.in);
		
		System.out.print("Yerden yüksekiği giriniz (m): ");
		h=scan.nextFloat();
		System.out.print("İlk hızını giriniz (m/s): ");
		vo=scan.nextFloat();
		
		t=Math.sqrt(2*h/9.8);
		x=vo*t;
		vy=9.8*t;
		v=Math.sqrt(Math.pow(vo, 2)+Math.pow(vy, 2));
		
		System.out.printf("\nCismin havada kalma süresi (s): %.3f",t);
		System.out.printf("\nCismin yatayda alacağı yol (m); %.3f",x);
		System.out.printf("\nCismin yere çarpma hızı (m/s): %.3f",v);


	}

}

File: Örnek-6.11/Python/yatayatıs.py
import math

h=float(input('Yerden yüksekliği giriniz (m): '))
vo=float(input('ilk hızını giriniz (m/s): '))

t=math.sqrt(2*h/9.8)
x=vo*t
vy=9.8*t
v=math.sqrt(math.pow(vo,2)+math.pow(vy,2))

print("\nCismin havada kalma süresi (s): %0.3f"%(t))
print("Cismin yatayda alacağı yol (m): %0.3f"%(x))
print("Cismin yere çarpma hızı (m/s): %0.3f"%(v))